Automattic/wp-calypso
# update pricing page for recommendations

#### Proposed Changes

(note this is a buddy session PR for @elliottprogrammer )

* Under certain circumstances, there is a plan recommendation section that previously displayed on the old pricing grid page that should also display on the new pricing page.  See 1202858161751496-as-1203255562458193 
* This PR adds the component to the new pricing page when the URL contains the compare_plans query string is present.
* The component itself will need to be updated in a separate PR to update the offerings it shows

![Screen Shot 2022-10-28 at 8 25 15 AM](https://user-images.githubusercontent.com/12895386/198586444-5d88e77d-be08-455d-bce1-5a757525700a.png)


#### Testing Instructions

* apply the patch and visit the following URL (substitute :site for a valid Jetpack site URL), verify the plan recommendation section shows up.  http://calypso.localhost:3000/plans/:site?compare_plans=jetpack_personal%2Cjetpack_backup_daily%2Cjetpack_anti_spam&view=bundles

#### Pre-merge Checklist






Related to #

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203255562458193
